strawberry/extensions/tracing/opentelemetry.py

```diff
@@ -76,6 +82,22 @@ def on_operation(self) -> Generator[None, None, None]:
         if not self._operation_name and self.execution_context.operation_name:
             span_name = f"GraphQL Query: {self.execution_context.operation_name}"
             self._span_holder[LifecycleStep.OPERATION].update_name(span_name)
+            self._span_holder[LifecycleStep.OPERATION].set_attribute(
+                graphql_attributes.GRAPHQL_OPERATION_NAME,
+                self.execution_context.operation_name,
+            )
+
+        # likewise for the operation type; we'll know it for sure after parsing.
+        # note that this means ``self.execution_context.operation_type`` must
+        # be kept in sync with ``graphql_attributes.GraphqlOperationTypeValues``.
```

Comment:
Generally speaking I'd prefer a mapper in these situations instead of a comment. However, the operation type is actually part of the GraphQL Spec, so it should be highly unlikely to go out of sync.  
I think you can drop the comment entirely 😊